### PR TITLE
Modifier texte du bundle de don

### PR DESCRIPTION
**Pull request by guillaumedenhez**

### CODE REVIEW

File: templates/explainer/article.html.twig [hunk 1]
```diff
@@ -94,7 +94,7 @@
         <section class="donate l__wrapper--narrow background--green b__nudge--bottom-huge">
             <div class="space--30 l__row l__row--h-stretch l__mobile--col">
                 <div class="donate__text">
-                    <h2 class="text--large">Donnez et <span class="text--on-white">faites gagner le progrès !</span></h2>
+                    <h2 class="text--large">Notre révolution démocratique a<span class="text--on-white">besoin de vous !</span></h2>
```
Comment (marineb) [inline templates/explainer/article.html.twig]: There's a space missing after the `a`.

File: templates/home/index.html.twig [hunk 2]
```diff
@@ -240,7 +240,7 @@
                 <div class="l__wrapper space--45-5 l__row l__row--h-stretch l__mobile--col">
                     <div class="donate__text text--white">
                         <h2 class="text--large">
-                            Notre révolution démocratique a <span class="text--on-pink">besoin de vous !</span>
+                            Notre révolution démocratique a<span class="text--on-pink">besoin de vous !</span>
```
Comment (marineb) [inline templates/home/index.html.twig]: Same here.

**Comment by marineb** [comment]
@Remg but isn't it weird that it's being checked-in / that a change is being recorded on it ?

**Comment by Remg** [comment]
@marineb yes, @guillaumedenhez changed some width for a specific class in this CSS file.

**Pull request merged**